src/main/resources/db/migration/V14__update_indexes.sql

```diff
@@ -0,0 +1,28 @@
+DROP INDEX idx_material_aid_offer_city;
+DROP INDEX idx_material_aid_offer_region;
+DROP INDEX idx_accommodation_offer_city;
+DROP INDEX idx_accommodation_offer_region;
+DROP INDEX idx_transport_offer_origin_city;
+DROP INDEX idx_transport_offer_origin_region;
+DROP INDEX idx_transport_offer_destination_city;
+DROP INDEX idx_transport_offer_destination_region;
+DROP INDEX idx_city_city;
+
+CREATE index idx_material_aid_offer_city ON material_aid_offer (lower(city));
+CREATE index idx_material_aid_offer_region ON material_aid_offer (lower(region));
+CREATE index idx_material_aid_offer_status ON material_aid_offer USING hash(status);
+CREATE index idx_material_aid_offer_user_id ON material_aid_offer (user_id);
+
+CREATE index idx_accommodation_offer_city ON accommodation_offer (lower(city));
+CREATE index idx_accommodation_offer_region ON accommodation_offer (lower(region));
+CREATE index idx_accommodation_offer_status ON accommodation_offer USING hash(status);
+CREATE index idx_accommodation_offer_user_id ON material_aid_offer (user_id);
+
+CREATE index idx_transport_offer_origin_city ON transport_offer (lower(origin_city));
+CREATE index idx_transport_offer_origin_region ON transport_offer (lower(origin_region));
+CREATE index idx_transport_offer_destination_city ON transport_offer (lower(destination_city));
+CREATE index idx_transport_offer_destination_region ON transport_offer (lower(destination_region));
+CREATE index idx_transport_offer_status ON transport_offer USING hash(status);
+CREATE index idx_transport_offer_user_id ON transport_offer (user_id);
+
+CREATE index idx_city_city ON city (lower(city));
```

Comment:
Actually, right now in the `CityRepository` we're not searching by `lower(city)` (but by `city`), and I think that this index is OK and we can add the case-insensitivity to the query, what do you think?

Right now all of the cities are written in lower-case, but we can't be sure it would always be the case.

It would require to change the query method in the `CityRepository`.

Reply:
I'm not sure if I changed it correctly. Can you take a look?

Reply:
I wasn't sure either just by looking at it, so I've added test for it - and it's OK :)